src/modes/edit_message.cc

```diff
@@ -95,6 +95,12 @@ namespace Astroid {
 
 # ifndef DISABLE_EMBEDDED
     embed_editor = !editor_config.get<bool> ("external_editor");
+    char const* xdg_session_type = std::getenv("XDG_SESSION_TYPE");
+    LOG (info) << "xdg_session_type: " << xdg_session_type;
+    if(xdg_session_type && std::string(xdg_session_type) == "wayland"){
+      LOG (warn) << "Embedded editor not available under wayland. Falling back to external";
```

Comment:
add note about: please check your configuration. Also I think this should be changed to error since the editor line specified by the user now probably is not a valid command.